Sources/Core/Microsoft.StreamProcessing/StringHandling/MultiString.cs

```diff
@@ -263,9 +267,8 @@ public static unsafe void AndEquals(ColumnBatch<long> left, ColumnBatch<long> ri
 
         #endregion
 
-        // Clone the multi-string shell only
         /// <summary>
-        /// Currently for internal use only - do not use directly.
+        /// Clone the multi-string shell only
```

Comment:
There are a few places here where the "do not use directly" comment is replaced by a different comment. I think the "do not use directly" directive doesn't change here - we're saying that this is a public method that we don't want public that that people shouldn't count on it being around. Perhaps combine the two comments?